src/services/ValidationService.ts

```diff
@@ -18,3 +26,49 @@ ValidationRules.customRule(
   (value) => Utils.isValidEmail(value, true),
   "Please enter a valid email address",
 );
+
+ValidationRules.customRule(
+  Validation.url,
+  (value) => Utils.isValidUrl(value),
+  "Please enter a valid url",
+);
+
+ValidationRules.customRule(
+  Validation.uniqueCollection,
+  (value) => Utils.isUniqueSimpleCollection(value),
+  "Please enter unique values",
+);
+
+ValidationRules.customRule(
+  Validation.imageUrl,
+  (value) => ImageService.isImageUrl(value),
+  "Please enter valid image URL",
```

Comment:
```suggestion
  "URL is not an image",
```
Reason: what's "valid"?

Reply:
"The URL does not point to an image"